### ISSUE TITLE
UI for Participate and Initiative pages

### ISSUE THREAD

**Issue by kristinashu** [mozilla/foundation.mozilla.org#1605]
Based on WIP wireframes in https://github.com/mozilla/foundation.mozilla.org/issues/1603 ([redpen](https://redpen.io/pw287b60c2a59295bc)) and https://github.com/mozilla/foundation.mozilla.org/issues/1604 ([invision](https://mozilla.invisionapp.com/share/GHL7FLR8R73#/screens/302107241_Proposal_3_-part-_-_Revised_-_Full_Hero)).

UI for these pages should be done in unison so they are cohesive. Trick will be balance our UI from our current site and Style Guide but with some deviations when necessary to make the site as sweet as possible!

@natalieworth @beccaklam @taisdesouzalessa please add any examples, ideas, benchmarks here!

**Comment by kristinashu** [comment]
Couple examples of editorial sites I like: https://www.good.is/ and https://slate.com/

**Comment by taisdesouzalessa** [comment]
For the time commitment label, I think Medium is a great example: https://medium.com/topic/technology - the labels are discreet yet useful and easy to find. I also like the little profile pictures on the bottom.

Charity: water is has a great page with good examples on how to display successes and wins - https://www.charitywater.org/projects/

**Comment by xmatthewx** [comment]
@natalieworth - for the participate page, let's change how pulse shows up at the bottom. Instead of a call to contribute to Pulse, let's show some projects and invite people in via help categories. This would be similar to the project cards that appear at the bottom of the page, but framed by the Help categories. 

Similar to the time commitment feature, Help categories give people a nice place to jump in. Some people want to offer code or design, others might not realize how helpful it can be to provide feedback or testing. We can link to individual projects and/or to help category lists on pulse. Example: https://www.mozillapulse.org/help/test-and-feedback

This staging entry lists all the [help categories](https://network-pulse-staging.herokuapp.com/entry/378).

Let's chat and maybe do some whiteboard sketches.

**Comment by beccaklam** [comment]
@kristinashu @natalieworth @taisdesouzalessa @sabrinang Made some changes based on crit yesterday: https://redpen.io/p/eh3a3293e829ff9f49
For the Participate page, since we'll most likely be using the card component, a way of adding visual interest could be through how we animate the cards coming onto the screen. Here's a pretty simple  benchmark: https://draft.co.jp/en/

**Comment by beccaklam** [comment]
@kristinashu @natalieworth @taisdesouzalessa Updates to the desktop UI are up on the redpen: https://redpen.io/p/eh3a3293e829ff9f49 plus mobile iterations. I'm missing a few things (time commitments on everything) but let me know how you think it's working. For the participate page, what if we just did the bands like mozilla.org, seems like at least half of people prefer that. We should probably decide soon though lol

**Comment by kristinashu** [comment]
I think you've nailed it! And once Nat's illustrations are added in, it will be amazing!

@jessevondoom can you take a look and answer a few of the outstanding questions in RedPen?

**Comment by kristinashu** [comment]
Excellent kick-off! 

Nat will be sharing two different illustration directions tomorrow(ish). Becca, please keep fine tuning and cleaning up the UI and then work with Gavin to see what he needs.

**Comment by beccaklam** [comment]
@kristinashu @taisdesouzalessa So I added some UI to the participate page. Not sure if we got rid of this idea or not but I remembered Jesse saying to tie the hero with its card below in of our critiques. So I just fleshed out that idea using the yellow from the CTA button on the hero and tying it with a yellow border on its card below: https://redpen.io/jv8a447745b88fc8ab Let me know what you think/if it's P2 if we still need this function.
